src/radiation/closure_m1.hpp

```diff
@@ -266,10 +264,12 @@ KOKKOS_FUNCTION M1Result ClosureM1<Vec, Tens2, SET>::SolveClosure(Real E, Vec co
   return result;
 }
 
-template <class Vec, class Tens2, class SET>
-KOKKOS_FUNCTION ClosureStatus ClosureM1<Vec, Tens2, SET>::M1Residuals(
-    const Real E, const Vec cov_F, const Real xi, const Real phi, const Vec con_tilg,
-    const Vec con_tild, Real *fXi, Real *fPhi) {
+template <class SET>
+KOKKOS_FUNCTION ClosureStatus ClosureM1<SET>::M1Residuals(const Real E, const Vec cov_F,
+                                                          const Real xi, const Real phi,
+                                                          const Vec con_tilg,
+                                                          const Vec con_tild, Real *fXi,
+                                                          Real *fPhi) {
```

Comment:
is this just clang format?

Reply:
Yeah, I didn't play any special formatting games here.